setup.py

```diff
@@ -12,9 +11,12 @@
         req for req in dev_requirements_file if not req.startswith("-")
     ]
 
+# get the version into a global variable named "version"
+exec(open("snowfakery/version.py").read())
```

Comment:
This is not the same approach that we ended up with in https://github.com/SFDO-Tooling/Snowfakery/pull/102. Probably makes sense to merge that one first, then update this branch.